lib/ezsession/classes/ezsession.php

```diff
@@ -261,16 +261,15 @@ static public function countActive()
      * {@link eZSession::start()}, so only call this if you don't start the session.
      *
      * @since 4.1
-     * @since v2018.06.0 Return false if session is already active since PHP 7.2 onwards emits a warning about this.
      * @return bool Depending on if eZSession is registrated as session handler.
     */
     static protected function registerFunctions( $sessionName = false, ezpSessionHandler $handler = null )
     {
-        if ( self::$hasStarted || self::$handlerInstance !== null || session_status() === PHP_SESSION_ACTIVE )
+        if ( self::$hasStarted || self::$handlerInstance !== null )
             return false;
 
         $ini = eZINI::instance();
-        if ( $sessionName !== false )
+        if ( $sessionName !== false && $sessionName !== session_name() )
```

Comment:
Alternative is to move all this session name logic into own method and only call it if session_status() !== PHP_SESSION_ACTIVE

But that said I'm ok with this, also, smaller diff.

Reply:
Good point.